### ISSUE TITLE
Not possible to add resources with HTTPS

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/input#6]
e.g. https xyz tiles or https wms layer

**Comment by wonder-sk** [comment]
Pending PR #22 - waiting for update of OSGeo4A docker container otherwise it would break builds

**Comment by wonder-sk** [comment]
@sklencar could you please test that a project with XYZ tile layer using HTTPS works now on Android? you can use e.g. this XYZ layer:
```
https://storage.googleapis.com/earthenginepartners-hansen/tiles/gfc2015/loss_year_new/{z}/{x}/{y}.png
```

**Comment by saberraz** [comment]
It should be working fine. The demo project is using https to access xyz files and they appear fine.

**Comment by wonder-sk** [comment]
oh so the previous android builds already include libcrypto and libssl? @sklencar

**Comment by wonder-sk** [comment]
fixed in #106
